src/bandersnatch/mirror.py

```diff
@@ -297,6 +303,9 @@ async def process_package(self, package: Package) -> None:
             return None
 
         # save the metadata before filtering releases
+        # (dalley): why? the original author does not remember, and it doesn't seem
+        # to make a lot of sense.
+        # https://github.com/pypa/bandersnatch/commit/2a8cf8441b97f28eb817042a65a042d680fa527e#r39676370
         if self.json_save:
```

Comment:
I agree we should only save JSON for mirror'ed packages based on filters. Happy for that PR. Would prefer it a dedicated PR tho.

Reply:
Yup, that would be no problem.  I just wanted to document first in case we ever ran across a reason to do so.